charts/ks-devops/values.yaml

```diff
@@ -1,14 +1,16 @@
 replicaCount: 1
 
 image:
+  # support 'ghcr.io/kubesphere-sigs' or 'kubespheresig'
+  registry: kubespheresig
```

Comment:
As we discussed in https://github.com/kubesphere-sigs/ks-devops/issues/61#issuecomment-882937563

I prefer to use `ghcr.io` as the default registry.

Normally, people read the usage from [README file](https://github.com/kubesphere-sigs/ks-devops/tree/master/charts/ks-devops). So, please add a description of this value into it.

Reply:
OK, I will add a description later